index.html

```diff
@@ -1 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute Page</title>
+</head>
+<body>
+    <Main id="main">
```

Comment:
Kindly use the correct case in semantic tags for html

Reply:
Please, always include an explanation, a "why", for each requested change because some reasons might seem obvious to you but not to the author of the code.

For example:

This is a well-explained request: _Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible_

This request lacks a "why": _Why don't you use an article tag here instead of a div?_

Reply:
I have updated the comment on this